pallets/summary/src/lib.rs

```diff
@@ -879,28 +873,17 @@ pub mod pallet {
         pub fn convert_data_to_eth_compatible_encoding(
             root_data: &RootData<T::AccountId>,
         ) -> Result<String, DispatchError> {
-            let eth_description =
-                EthAbiHelper::generate_ethereum_description_for_signature_request(
-                    &T::AccountToBytesConvert::into_bytes(
-                        root_data
-                            .added_by
-                            .as_ref()
-                            .ok_or(Error::<T>::CurrentSlotValidatorNotFound)?,
-                    ),
-                    &EthTransactionType::PublishRoot(PublishRootData::new(
-                        *root_data.root_hash.as_fixed_bytes(),
-                    )),
-                    match root_data.tx_id {
-                        None => EMPTY_ROOT_TRANSACTION_ID,
-                        _ => *root_data
-                            .tx_id
-                            .as_ref()
-                            .expect("Non-Empty roots have a reserved TransactionId"),
-                    },
-                )
-                .map_err(|_| Error::<T>::InvalidRoot)?;
+            let root_hash = *root_data.root_hash.as_fixed_bytes();
```

Comment:
Why do we need this function in this pallet if all the ethereum logic is handled by eth bridge? I think we can remove it together with the ecdsa signature logic here.

Reply:
we are using this function for the approving of the root hash before it being published to ethereum, aren't we?

Reply:
I don't think we need ecdsa signatures for approving roots, the standard signature (SR25519) should be enough for that. And IIRC, we already do that for the unsigned extrinsics.